app/collections/show.cjsx

```diff
@@ -8,11 +8,15 @@ Avatar = require '../partials/avatar'
 Loading = require '../components/loading-indicator'
 HandlePropChanges = require '../lib/handle-prop-changes'
 TitleMixin = require '../lib/title-mixin'
+classNames = require 'classNames'
 
 counterpart.registerTranslations 'en',
   collectionPage:
     settings: 'Settings'
     collaborators: 'Collaborators'
+    collectionsLink: '%(user)s\'s\u00a0Collections'
+    favoritesLink: '%(user)s\'s\u00a0Favorites'
+    userLink: '%(user)s\'s\u00a0Profile'
```

Comment:
No, but since word order varies between language, the best way to globalize a message involving the username is to pass it in to the full message so it can be correctly positioned.

Consider French for example: "Les collections de Alex"

The alternative would be to only translate part of a sentence and encode the word order algorithmically - which I am sure you can see would be a bad idea.